### PR TITLE
Bug 1260240 fixed info on creating node config files

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1260240

I added the cert options to the example command. Also, did some rewording.

@DirectXMan12 can I get an ack this is what you were explaining to me over email? Thanks!

cc: @rjhowe

### CODE REVIEW

File: install_config/master_node_configuration.adoc [hunk 1]
```diff
@@ -960,15 +961,22 @@ related files in the specified directory:
...
-For the `--hostnames` option in the above command, use a comma-delimited list of
-every host name or IP address you want server certificates to be valid for. The
-above command also assumes that certificate files are located in an
-*_openshift.local.config/master/_* directory. If they are not, you can include
-options to specify their location. Run the command with the `-h` option to see
-details.
+When creating node configuration files, the `--hostnames` option accepts a
+comma-delimited list of every host name or IP address you want server
+certificates to be valid for. The above command also assumes that certificate
+files are located in an *_openshift.local.config/master/_* directory. If they
```
Comment (DirectXMan12) [inline install_config/master_node_configuration.adoc]: not sure this line is needed any more, since we explicitly specify the path to the certs above

**Review by DirectXMan12 — CHANGES_REQUESTED** [review]
minor nit, otherwise I think this looks good

**Comment by bfallonf** [comment]
Thanks @DirectXMan12 . I've made the suggested change.

@ahardin-rh @bmcelvee @adellape Peer review please?

File: install_config/master_node_configuration.adoc [hunk 2]
```diff
@@ -931,11 +931,13 @@ to limit filesystem permissions and access to the key file.
 [[creating-new-configuration-files]]
 
 == Creating New Configuration Files
-For masters, the `openshift start` command accepts options that indicate that it
-should simply write the configuration files that it would have used, then
-terminate. For nodes, a configuration file can be written using the `oadm
-create-node-config` command. Creating new configuration files is useful to get a
-starting point for defining your configuration.
+
+Creating new configuration files is useful to get a starting point for defining
```
Comment (ahardin-rh) [inline install_config/master_node_configuration.adoc]: This sentence is a little awkward to me. Maybe we can edit it to something like the following? 
To define your configuration, start by creating new configuration files.

**Comment by ahardin-rh** [comment]
Just one comment from me, otherwise looks good! :star2:

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/master_node_configuration.adoc#install-config-master-node-configuration[Aggregating Container Logs]
|Fixed the options for creating a configuration file in the xref:../install_config/master_node_configuration.adoc#creating-new-configuration-files[Creating New Configuration Files] section.
%

**Pull request merged**